unittests/test_sanity_functions.py

```diff
@@ -340,6 +340,12 @@ def test_assert_reference(self):
         self.assertTrue(sn.assert_reference(-0.9, -1, upper_thres=0.1))
         self.assertTrue(sn.assert_reference(-0.9, -1))
 
+        # Check upper threshold values greater than 1
+        self.assertTrue(sn.assert_reference(30.0, 10.0, None, 3.0))
+        self.assertTrue(sn.assert_reference(-50.0, -20.0, -2.0, 0.5))
```

Comment:
I don't understand why this should succeed. The value should be in [-40, -10] and it's -50.

Reply:
Because in this case -2.0 means -200 % difference thus the corresponding lower bound is -20.0 -2.0 * 20.0 = -60.0.